pypeit/spectrographs/keck_deimos.py

```diff
@@ -1127,6 +1292,10 @@ def get_maskdef_slitedges(self, ccdnum=None, filename=None, debug=None):
         if ccdnum is None:
             msgs.error('A detector number must be provided')
 
+        # parse ccdnum
+        nimg, _det = self.validate_det(ccdnum)
+        _ccdnum = [d for d in _det]
```

Comment:
Does this mean `_ccdnum` and `_det` are identical?  Could you just set `_ccdnum` directly: `nimg, _ccdnum = self.validate_det(ccdnum)`?